test/mlir/test/gc/Dialect/CPURuntime/cpuruntime-to-llvm.mlir

```diff
@@ -1,6 +1,10 @@
 // RUN: gc-opt %s --convert-cpuruntime-to-llvm | FileCheck %s
 
 module {
+  // CHECK: llvm.func @gcThreadAlignedFree(!llvm.ptr)
```

Comment:
suggest to add an integration test like we did for cpurunner printf tests, to see if it runs well.

Reply:
added a test in cpurunner/allocators.mlir.